internal/rollup/rollup_bundle.bzl

```diff
@@ -334,6 +334,9 @@ def _run_terser(ctx, input, output, map_output, debug = False, comments = True,
         args.add("--debug")
         args.add("--beautify")
 
+    # we need this arg to enable worker_threads
+    args.add("--node_options=--experimental-worker")
```

Comment:
If on Node 12+ this shouldn't be added

Reply:
Fair, not sure how to get the nodejs version in in skylark though.
@alexeagle You mentioned there's a way to do it the other day?

Reply:
Once we have toolchains it would actually be possible to add the node version to the toolchain and then one could get it in starlark. Not sure though how it would be possible currently though.

Reply:
If problematic, node 12 currently won't error with the option present.  So this could be addressed moving forward.